### DISCUSSION TITLE
Per-app notifications

### DISCUSSION THREAD

**Discussion by Bastih18** [coollabsio/coolify#4569] [Improvement Requests]
Consider adding support for per-app notifications, such as the ability to configure Discord Webhooks for specific apps. This would allow users to receive deployment notifications or logs for individual apps, rather than logging all activity across Coolify to a single webhook.

This feature would be particularly useful for scenarios where users want to limit notifications to a specific app. For example, they might only want deployment logs for a particular app to be shared in a public Discord server, without exposing logs or updates for other projects.

**Comment by Natlink** [comment]
This would be really great to allow notification per project.
For exemple, if we have a coolify server which host multiple projects and there is multiple different teams working on each of these project. When App1 is deployed, the team of App2 doesn't care about it.
Per project notification would let us dispatch the useful information only to people who needs it.

As a temporary workaround, it's possible to setup a webhook call in the dockerfile of the deploy for exemple, or use post-deployment command to call a manually crafted notification script. but it's far to be a good thing for production environment IMO

**Comment by 50l3r** [comment]
It would also be a good idea to have a webhook as a notification, or at least temporarily.

For a note, you could use a service like [shoutrrr](https://containrrr.dev/shoutrrr/v0.8/), which gives you even more versatility than a standard webhook.

Then, in your webhook, you can use logic to separate notifications:

If project is equal to `Project 1` -> Do something
If project is equal to `Project 2` -> Do something but different

**Comment by ToreAndreRosander** [comment]
This feature requests should be implemented, It's strange that it's not possible to filter the notifications.

**Comment by pergpau** [comment]
This would be a great feature 🥇 Notifications aren't really that useful on a server level, everything is all about the projects imo.
